app/grandchallenge/cases/image_builders/metaio_utils.py

```diff
@@ -55,7 +55,12 @@
 
 STUDYDATE_MATCH_REGEXP: Pattern = re.compile(r"^\d{4}\d{1,2}\d{1,2}$")
 
+PIXELSPACING_MATCH_REGEXP: Pattern = re.compile(r"^(\d*\.\d+|\d+)\\(\d*\.\d+|\d+)$")
```

Comment:
This regex does not match the definition of DS: http://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_6.2.html

`[-+]?\d*\.?\d+([eE][-+]?\d+)?\\[-+]?\d*\.?\d+([eE][-+]?\d+)?` is probably what you want.

Also, you need to run `black` on this file to format the code correctly.

Reply:
Okay, I just pushed a new change before reading your reply, I will format the code correctly next time.

Reply:
The test just failed, I will change the pattern to the one you suggested. It seems that I looked at the wrong error log before writing my new pattern.

Reply:
It finally works, all checks have passed. Thanks for the help!